lib/Service/AutoConfig/MxRecord.php

```diff
@@ -56,4 +56,17 @@ public function query(string $host) {
 		// TODO: sort by weight
 		return $mxRecords;
 	}
+
+	public function stripSubdomain(string $domain): string {
```

Comment:
I suggest to make the method private. We can use `$this->invokePrivate($object, 'methodName');` in a test case to execute private methods.